ui/src/app/course/components/editor/editor.component.ts

```diff
@@ -39,13 +45,23 @@ export class CourseEditorComponent implements OnInit, OnDestroy {
 		if (this._isEditMode()) {
 			this._startLoading();
 
-			this.subs.push(
-				this._courseService.getCourse(this.courseId)
-					.subscribe((course: TCourse) => this._setCourse(course), this._handleNotFound.bind(this))
-			);
+			this._store.pipe(select(getCourseDetail)).subscribe((course: TCourse) => {
+				this._setCourse(course);
+			});
+			this._loadCourse();
 		}
 	}
 
+	private _loadCourse() {
+		this.subs.push(
```

Comment:
When a course if opening there 2 the same GET requests (and 2 OPTIONS one for each GET)
![image](https://user-images.githubusercontent.com/81073/72929561-22a11b00-3d6b-11ea-9414-eab27fee9b1c.png)

Reply:
now the bug does not play